src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/EnterBreathingSpaceCuiCallbackHandler.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.reform.civil.handler.callback.user;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackResponse;
+import uk.gov.hmcts.reform.civil.callback.Callback;
+import uk.gov.hmcts.reform.civil.callback.CallbackHandler;
+import uk.gov.hmcts.reform.civil.callback.CallbackParams;
+import uk.gov.hmcts.reform.civil.callback.CaseEvent;
+import uk.gov.hmcts.reform.civil.model.BusinessProcess;
+import uk.gov.hmcts.reform.civil.model.CaseData;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_START;
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.ABOUT_TO_SUBMIT;
+import static uk.gov.hmcts.reform.civil.callback.CallbackType.SUBMITTED;
+import static uk.gov.hmcts.reform.civil.callback.CaseEvent.ENTER_BREATHING_SPACE_CUI;
+
+@Service
+@RequiredArgsConstructor
+public class EnterBreathingSpaceCuiCallbackHandler extends CallbackHandler {
+
+    private static final List<CaseEvent> EVENTS = Collections.singletonList(CaseEvent.ENTER_BREATHING_SPACE_CUI);
+
+    private final ObjectMapper objectMapper;
+
+    @Override
+    public List<CaseEvent> handledEvents() {
+        return EVENTS;
+    }
+
+    @Override
+    protected Map<String, Callback> callbacks() {
+        return Map.of(
```

Comment:
could have a map as a final field. so that every time we call a method it does not create a new map

Reply:
For this one, I have tried to change but not successful, there is error after changing protected to final. Also, I have checked other handler and found that they are similar to the one that I am using

Reply:
like this Kenneth for example: 
![image](https://github.com/hmcts/civil-service/assets/95431510/cab76493-0a0a-4e39-9b81-f26f71abacaa)